cmd/config.go

```diff
@@ -157,6 +174,33 @@ func RunPhases(phase string, v reflect.Value) {
 	}
 }
 
+// ReloadServices iterates through key/values calling reload on applicable services.
+func ReloadServices(v reflect.Value) {
+	for i := 0; i < v.NumField(); i++ {
+		if reflect.Value.IsZero(v.Field(i)) {
+			continue
+		}
+
+		var err error
+		switch v.Field(i).Kind() {
+		case reflect.Slice:
+			for j := 0; j < v.Field(i).Len(); j++ {
+				serviceItem := v.Field(i).Index(j).Interface()
+				switch c := serviceItem.(type) {
+				case Reloader:
```

Comment:
Add a comment here please,
Suggestion:
"Check to see if the selected type field is of type Reloader interface, meaning reload can be called"